graphqlhub-schemas/src/github.js

```diff
@@ -100,8 +151,8 @@ let IssueLabelType = new GraphQLObjectType({
   }
 });
 
-let grabUsernameAndReponameFromURL = (url) => {
-  let array = url.split('/repos/')[1].split('/issues')[0].split('/');
+let grabUsernameAndReponameFromURL = (url, resource) => {
+  let array = url.split('/repos/')[1].split('/' + resource)[0].split('/');
```

Comment:
thanks for fixing this - can we make it possible without having to pass in the additional argument?

I think something like this works:

```
  let array = url.split('https://api.github.com/repos/')[1].split('/');
  return {
    username : array[0],
    reponame : array[1],
  };
```